sagemaker/launch/launch_utils.py

```diff
@@ -238,3 +239,33 @@ def check_name_format(name_str):
                                 'expression pattern: ' + \
                                 r'^[a-zA-Z0-9]([\-a-zA-Z0-9]*[a-zA-Z0-9])$.')
     return name_str
+
+
+def has_tokenize_transformation(graph_config):
+    """ A simple sanity check of tokenize_hf transformation in graph construction JSON file
+    
+    This function only check if there is a tokenize_hf feature transformation. For a
+    comprehensive, please use the `graphstorm.dataloading.metadata.config_json_sanity_check`
+    function.
+    Because we design this sagemaker package to be independent to the core code of graphstorm,
+    this function does not import the config_json_sanity_check function, but implements a
+    simple and quick check for the tokenize transformation.
+    
+    Parameters
+    ----------
+    graph_config: JSON object
+        A JSON object from a graph construciton JSON file.
+
+    Return
+    ------
+    has_tokenize_transform: bool
+        A boolean value. If the JSON oject contains at least one tokenize transformation, return
+        True, otherwise False.
+    """
+    tokenize_str = 'tokenize_hf'
+    # Convert json object to string, and then use string search
+    json_str = json.dumps(graph_config)
+    has_tokenize_transform = tokenize_str in json_str
```

Comment:
This is a fragile implementation, we could have `tokenize_hf` in other parts of the config, e.g. a path or feature name. 

It's better to walk the JSON structure and specifically look for the key/value  `"name": "tokenize_hf"` within feature definitions.

Reply:
If walk through the JSON structure, it will be same as the GraphStorm core code in `metadata.py`. Better not to duplicate the same function for only one field.

To make sure `tokenize_hf` is in transformation field, new implementation use regex instead of a simple `in` function.